CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# 1.0.10.4 (2020-12-18)
+## Edge Agent
+### Bug Fixes
+* Fix vulnerability issues in ARM-based docker images [6603778](https://github.com/Azure/iotedge/commit/6603778d2c4dbb63aea4627f66feb5920eede3c6)
+* GetModuleLogs works with http endpoints [cf176be](https://github.com/Azure/iotedge/commit/cf176be66bab28b032f0069a445ed21cc9fcdfb9)
+
+## Edge Hub
+### Bug Fixes
+* Fix `edgehub_queue_length` metric [2861dbc](https://github.com/Azure/iotedge/commit/2861dbc63757787ef52897d9acc732430b46313b)
```

Comment:
It seems it is missing some commits: 
- darobs merge -- not sure how to summarize it
- like sha256 thumbprint support
- fix clean processor